test/middleware/getPhotos.js

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
Rename of getImages test with additional work for assertions. See above comment re: mocks for my feelings.